.travis.yml

```diff
@@ -13,6 +13,10 @@ notifications:
   email: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("Mocking"); Pkg.test("Mocking"; coverage=true)';
+  - |
+    julia -e '
+      VERSION >= v"0.7.0-DEV.3656" && using Pkg
+      VERSION >= v"0.7.0-DEV.5183" ? Pkg.instantiate() : Pkg.clone(pwd())
```

Comment:
What's the difference between `Pkg.instantiate()` and `Pkg.develop(pwd())`?

Reply:
I originally used `develop` but I believe that resulted in a UUID error. Doing a quick experiment here: https://travis-ci.org/invenia/Mocking.jl/builds/397006755

Reply:
Results in:
```julia
ERROR: Cannot add package with the same name or uuid as the project
```

Reply:
I guess that only works if the package doesn't have a project file.

Reply:
I'll run another test to see if `Pkg.instantiate()` works on packages without project files: https://travis-ci.org/invenia/Mocking.jl/builds/397024900

Reply:
Looks like `Pkg.instantiate()` works on packages with or without Project.toml files.

Reply:
"Idiomatic" would be something like:

```
if VERSION >= v"$PKG3_EXISTS"
    Pkg.build(); Pkg.test(;coverage = true)
else
    Pkg.clone(pwd()); Pkg.build("Mocking"); Pkg.test("Mocking"; coverage=true)'
end
```